src/Connector.sol

```diff
@@ -125,8 +128,25 @@ contract CentrifugeConnector {
         // TODO: send message to the gateway. Depends on https://github.com/centrifuge/connectors/pull/52
     }
 
+    function decreaseInvestOrder(uint64 poolId, bytes16 trancheId, uint128 amount) public {
```

Comment:
@offerijns @mustermeiszer I am not sure what the logic should be for `Increase/DecreaseRedeemOrder`, `CollectRedeem`, CollectInvest`, etc